.circleci/config.yml

```diff
@@ -29,16 +29,19 @@ executors:
   node-10:
     <<: *defaults
     docker:
-      - image: circleci/node:10.20.1
+      - image: cimg/node:10.20.1
```

Comment:
Why the change from `circleci` to `cimg`?

Reply:
![image](https://user-images.githubusercontent.com/58703698/118043111-eec47d00-b342-11eb-9860-6aa9c14c6a00.png)